apps/builder/app/builder/features/style-panel/shared/property-name.tsx

```diff
@@ -180,13 +183,17 @@ export const PropertyName = ({
 
   return (
     <Flex align="center">
-      <Tooltip
-        content={label}
-        delayDuration={600}
-        disableHoverableContent={true}
-      >
-        {labelElement}
-      </Tooltip>
+      {typeof label === "string" ? (
+        <Tooltip
+          content={label}
+          delayDuration={600}
+          disableHoverableContent={true}
+        >
+          {labelElement}
+        </Tooltip>
+      ) : (
+        labelElement
+      )}
```

Comment:
If this is easy to do, I'd add `tooltip?: string` to `PropertyNameProps` and would render tooltip based on this property. 

But if hard for some reason, we can fix this later because we plan to change these tooltips later anyway AFAIK.

Reply:
I will add comment why no tootip here needed